src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const repairedRobot = {};
+
+  for (const key in robot) {
+    if (repairedRobot.hasOwnProperty([robot[key]])) {
+      return null;
+    } else {
+      repairedRobot[robot[key]] = key;
+    }
```

Comment:
```suggestion
    }
    repairedRobot[robot[key]] = key;
```

No need for this `else` because if the first check passes, the function will return 'null' and stop, after that no further conditions won't execute